Spring-Boot/Spring-Boot-Academy/pom.xml

```diff
@@ -66,6 +66,11 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-csv</artifactId>
+            <version>1.8</version>
```

Comment:
Don't put the version here, please specify the version in the `Dependencies/pom.xml`